client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1431,6 +1431,8 @@
             },
             templateUrl: 'scripts/superdesk-authoring/views/send-item.html',
             link: function sendItemLink(scope, elem, attrs) {
+                var parentScope = scope.$parent.$parent;
```

Comment:
If I understand correctly, this grandparent scope is only referenced to obtain a reference to the `monitoring` controller used by the `MonitoringViewDirective`? If so, this pattern should be avoided, because it is fragile and sensitive to changes in the scope hierarchy.

The right way to do it is to declare that this directive depends on the parent `MonitoringViewDirective` (`require: '^sdMonitoringView'`), and then the `monitoring` controller will be passed to the `link()` function as its 4th argument:

```js
function sendItemLink(scope, elem, attrs, ctrl) {
    // ctrl === MonitoringController
```